_config.yml

```diff
@@ -16,9 +16,9 @@ plugins:
 # Note the collection name is available from the page, e.g. page.collection
 # Thus you can do site[page.collection] to access the collection
 collections:
-  docs: # unversioned docs
```

Comment:
I never liked this separate docs section for the meta stuff so I duplicated it and put it in both v4 and v5